pkgs/top-level/all-packages.nix

```
@@ -24191,6 +24191,10 @@ in
 
   wmctrl = callPackage ../tools/X11/wmctrl { };
 
+  wmderland = callPackage ../applications/window-managers/wmderland { };
+
+  wmderlandc = callPackage ../applications/window-managers/wmderland/ipc-client.nix {};
```

```suggestion
  wmderlandc = callPackage ../applications/window-managers/wmderlandc {};
```

The files don't share anything major so we should do this.

---

> The files don't share anything major so we should do this. This would also require to move and rename the nix file.

Thank you. I've fixed it.

> Also wmderlandc builds on darwin but wanderland not:

Thank you again for your reviewing.
I finally found that that errors caused by the link-time optimization flag (`-flto`) defined in CMakeLists.txt. (https://github.com/aesophor/wmderland/blob/a40a3505dd735b401d937203ab6d8d1978307d72/CMakeLists.txt#L13)

Hence, I must remove this linker option when the package is built on darwin. But I am not so clear how it's done. Could you give me an idea?

---

I think that one possible solution is to write a patch to remove that option and to fetch (or placing the patch in the nixpkgs repository and loading it directly) and to apply it when the building platforms matches with darwin. How do you feel about it?